src/integrationTest/java/uk/gov/hmcts/reform/divorce/callback/SolicitorLinkCaseCallbackTest.java

```
@@ -56,7 +61,13 @@ public void givenAosAwaitingState_whenSolicitorLinksCase_thenCaseShouldBeLinked(
 
         String caseId = String.valueOf(caseDetails.getId());
         updateCaseForCitizen(caseId, null, TEST_AOS_STARTED_EVENT_ID, petitionerUserDetails);
-        updateCase(caseId, null, AOS_NOMINATE_SOL_EVENT_ID, ImmutablePair.of(AOS_LETTER_HOLDER_ID, pinResponse.getUserId()));
+        updateCase(caseId, null, AOS_NOMINATE_SOL_EVENT_ID,
+                ImmutablePair.of(D8_RESPONDENT_SOLICITOR_NAME, TEST_SOLICITOR_NAME),
+                ImmutablePair.of(D8_RESPONDENT_SOLICITOR_EMAIL, TEST_SOLICITOR_EMAIL)
+        );
```

Those fields are required - the test was working before because the email generation failure was ignored as the callback for `AOS_NOMINATE_SOL_EVENT_ID` was in the submitted stage instead of before submit